packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -218,6 +221,7 @@ class AppSizeController {
 
     // Build a tree with [TreemapNode] from [processedJsonMap].
     final newRoot = generateTree(processedJson)!;
```

Comment:
This method is only invoked when we first load the JSON file into DevTools, right? If that's the case, let's rename `newRoot` to `appRoot` just to make it more clear.